classes/class-wc-rest-connect-shipping-label-controller.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+if ( class_exists( 'WC_REST_Connect_Shipping_Label_Controller' ) ) {
+	return;
+}
+
+class WC_REST_Connect_Shipping_Label_Controller extends WP_REST_Controller {
+
+	/**
+	 * Endpoint namespace.
+	 *
+	 * @var string
+	 */
+	protected $namespace = 'wc/v1';
+
+	/**
+	 * Route base.
+	 *
+	 * @var string
+	 */
+	protected $rest_base = 'connect/shipping-label';
+
+	/**
+	 * Register the routes for order notes.
+	 */
+	public function register_routes() {
+		register_rest_route( $this->namespace, '/' . $this->rest_base . '', array(
+			array(
+				'methods'             => WP_REST_Server::EDITABLE,
+				'callback'            => array( $this, 'update_items' ),
+				'permission_callback' => array( $this, 'update_items_permissions_check' ),
+			),
+		) );
+	}
+
+	private function validation_error( $fields ) {
+		return new WP_Error('validation_failed',
+			'Error!!!',
```

Comment:
Let's go ahead and at least `__( 'TODO Error', 'woocommerce' )` this now. If we get too many of these we're going to miss something eventually :)